modules/face/README.md

```diff
@@ -1,8 +0,0 @@
-Face recognition techniques
-===========================
-
-Collection of face recognition techniques:
-
-1. Eigen Faces
-2. Fisher Faces
-3. Local Binary Pattern Histograms
```

Comment:
Instead of removing the README, it should be updated to also contain the stuff you are going to make. But in principle this file should not be changed until the moment we actually finish this up.